tests/v2/specs/petstore.yaml

```diff
@@ -596,9 +596,9 @@ definitions:
         type: "string"
         description: "Order Status"
         enum:
-          - "placed"
-          - "approved"
-          - "delivered"
```

Comment:
Thanks so much for adding this in a test, but would you mind instead adding unit tests to:
- `tests/v2/index.test.ts`
- `tests/v3/index.test.ts`

This `petstore.yaml` is a direct copy-paste of Swagger’s example that’s not meant to be edited, and these snapshots are so easy to blow away and lose this test in the process. This is more of a “pass/fail” test than testing specific behavior like what you added—that’s what those spec tests are for.